packages/console/src/console-history/ConsoleHistoryItemTooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { type ReactElement } from 'react';
+import { Content, Heading } from '@deephaven/components';
+import { type ConsoleHistoryActionItem } from './ConsoleHistoryTypes';
+import { getTimeString } from './ConsoleHistoryItemUtils';
+
+interface ConsoleHistoryActionItemTooltipProps {
+  item: ConsoleHistoryActionItem;
+}
+
+function CommandHistoryItemTooltip(
+  props: ConsoleHistoryActionItemTooltipProps
+): ReactElement {
+  const { item } = props;
+  const { startTime, serverStartTime, serverEndTime } = item;
+  const endTime = item.endTime ?? Date.now();
```

Comment:
Should memoize `endTime`, otherwise if this re-renders the date will update.
Could memoize the component as well.